template/php/function/php-extension.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+echo "Installing PHP extensions"
+# Add your extensions in here
```

Comment:
Add some text here explaining that you can use the helpers from the Docker library PHP image, and that its also possible to `apk` add stuff here as well where you have lib dependencies.